Source/SwiftLintFramework/Rules/Lint/UnusedCaptureListRule.swift

```diff
@@ -0,0 +1,128 @@
+import Foundation
+import SourceKittenFramework
+
+public struct UnusedCaptureListRule: ASTRule, ConfigurationProviderRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static var description = RuleDescription(
+        identifier: "unused_capture_list",
+        name: "Unused Capture List",
+        description: "Unused reference in a capture list should be removed.",
+        kind: .lint,
```

Comment:
can you add `swiftVersion: .fourDotTwo`? `SwiftSyntaxKind.closure` is only returned by SourceKit in 4.2+